### DISCUSSION TITLE
Congrats on Version 3 - one concern

### DISCUSSION THREAD

**Discussion by osseonews** [oramasearch/orama#827] [General]
Congrats on Version 3. You have created a really amazing product here. Going to test it again on a new project we are doing where the use case fits exactly what this product offers. My only concern is that the AI chat is very slow sometimes. I tried it on your partner websites and many times the "...." will flash for over a minute and nothing will happen. Had it on your site also when asking the "Shopify" question. It's fast after a question is answered, but sometimes very slow on the initial question.

**Comment by micheleriva** [comment]
Hi there! Thank you so much! On Orama Cloud, we're currently using OpenAI... and it's not a good day for their reliability: https://status.openai.com.

<img width="894" alt="Screenshot 2024-10-17 at 23 44 51" src="https://github.com/user-attachments/assets/a1ff847b-4e6c-4bc8-bcab-555715821a66">

We're putting in place some strategy to route to a different provider when that happens.

**Comment by osseonews** [comment]
OK, thanks for the update, didn't realize openai had problems today. Also, quick question, but do you have any plans to add image search and AI to Orama? Right now it's all textual, but I'm wondering if you can create indexes of images at some point, so you can run AI on them to search for similarities. Thanks.
